routersploit/modules/libs/lzs.py

```diff
@@ -0,0 +1,140 @@
+# !/usr/bin/env python
```

Comment:
Anything in the `modules` directory will be available for completer. 
```
rsf > use 
creds     exploits  scanners   libs
rsf > use 
```
This cause build fail. There is nothing we can do with `/libs` within interpreter session so please move this file to `routersploit/utils.py`